### PR TITLE
Validated MicrosoftGesturesInstallDir env variable before each projec…

### PR DESCRIPTION
**Pull request by mosdav**
…t build

### CODE REVIEW

File: common.targets [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="ValidateMicrosoftGesturesInstallDirEnvVaraibleIsDefined" BeforeTargets="PreBuildEvent">  
+        <Error Text="MicrosoftGesturesInstallDir environment variable is not defined ! kindly install Microsoft Gestures SDK from https://aka.ms/gestures/download and restart this VisualStudio instance"
```
Comment (YoniSmolin) [inline common.targets]: 1. No need for a space before an exclamation mark
2. Add a period at the end of the second sentence
3. "this VisualStudio instance" → "Visual Studio" (in my subjective opinion, it's clearer that way - feel free to override)
4. "kindly" → "Kindly"

File: common.targets [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="ValidateMicrosoftGesturesInstallDirEnvVaraibleIsDefined" BeforeTargets="PreBuildEvent">  
+        <Error Text="MicrosoftGesturesInstallDir environment variable is not defined ! kindly install Microsoft Gestures SDK from https://aka.ms/gestures/download and restart this VisualStudio instance"  
+               Condition="'$(MicrosoftGesturesInstallDir)' == ''" />
+		<Error Text="MicrosoftGesturesInstallDir environment variable points to non existent directory ,$(MicrosoftGesturesInstallDir), ! kindly install Microsoft Gestures SDK from https://aka.ms/gestures/download and restart this VisualStudio instance"
```
Comment (YoniSmolin) [inline common.targets]: 1. Same comments as above
2. The second comma is unnecessary
3. I'd forgo using commas here altogether and opt for single\double quotation marks

File: common.targets (hunk 1 above)
Comment (mosdav) [reply common.targets]: Fixed

File: common.targets (hunk 2 above)
Comment (mosdav) [reply common.targets]: Fixed

**Comment by mosdav** [comment]
@YoniSmolin  I've incorporated your phrasing feedback - please re-review

**Pull request merged**